lms/djangoapps/courseware/management/commands/dump_course_ids.py

```diff
@@ -35,7 +35,6 @@ def handle(self, *args, **options):
 
         if store is None:
             raise CommandError("Unknown modulestore {}".format(name))
+        output = u'\n'.join(unicode(course.id) for course in store.get_courses()) + '\n'
 
-        output = u'\n'.join(course.id.to_deprecated_string() for course in store.get_courses()) + '\n'
-
-        return output.encode('utf-8')
+        return output
```

Comment:
I think this would be cleaner as a loop that calls self.stdout.write, but we can merge anyway.